R/wwinference.R

```diff
@@ -375,17 +416,68 @@ fit_model <- function(compiled_model,
 get_mcmc_options <- function(
```

Comment:
as discussed, remove causes where we use the `cmdstanr` defaults and include only cases where we overwrite it. 

Also need to consider whether you expect the user ever to call this function, or if it's just a way of documenting the defaults.

Reply:
I think I will change this to be an internal function

Reply:
One problem I see with making it internal is that it makes the documentation less obvious. I think for now for simplicity of documentation I'd leave it exported unless you want to get rid of `get_mcmc_options()` entirely. In the latter case I'd suggest moving the small number of non-`cmdstanr-`default args configured in `get_mcmc_options()` to become top-level keyword arguments to `wwinference()` (i.e. you can do:

```R
wwinference(
    <arguments>,
    chains = 3)
```